src/test-support.js

```diff
@@ -36,12 +36,16 @@ function jscodeshiftTest(options) {
         let testName = filename.replace(`.input${extension}`, '');
         let inputPath = path.join(details.fixtureDir, `${testName}.input${extension}`);
         let outputPath = path.join(details.fixtureDir, `${testName}.output${extension}`);
+        let optionsPath = path.join(details.fixtureDir, `${testName}.options.json`);
+        process.env.CODEMOD_CLI_ARGS = fs.pathExistsSync(optionsPath)
```

Comment:
@rwjblue This does not seem to be working. How do we set the env params in this case?

Reply:
@rwjblue Now it is working, the issue was the options were set before the tests started. So in case of multiple tests back to back the last set env variables were available for all tests. Added beforeEach and afterEach hooks to set and reset the env vars.